dev/fattest.simplicity/bnd.bnd

```diff
@@ -56,6 +56,9 @@ Private-Package: \
 
 test.project: true
 
+# Allow access to @Repeatable annotation from Java 8, while still compiling for Java 7 compatibility
+-buildpath.bootclasspath: ${javac.bootclasspath.1.8}
```

Comment:
This gives the simplicity framework more than just access to `@Repeatable`, and I don't think it's worth the risk for FATs that still need to run on Java 7

Reply:
Removed this